package.json

```diff
@@ -4,11 +4,12 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test:dbscript": "cross-env NODE_ENV=test sequelize db:migrate",
-    "server": "nodemon --watch src --exec babel-node src/index.js",
-    "start": "babel-node src/index.js",
+    "db:migration": "cross-env NODE_ENV=test sequelize db:migrate",
+    "undo:migration": "cross-env NODE_ENV=test sequelize db:migrate:undo:all",
+    "server": "nodemon --watch src --exec babel-node src/server.js",
```

Comment:
I believe that you set up `db:migration` and  `undo:migration` command for the test script and it is not included.

Reply:
Please do git pull origin ft-login-user-167735036 to ensure you are working with the current update and undo any previous migration by using npm run undo:migration and re-run the migration by using npm run db:migration, one last thing make sure your test database is setup to something like this DATABASE_URL_TEST=postgres://postgres:password@localhost:5432/icemantest and you have to create the test database manually while for the swagger you can test the login endpoint under auth section but it return 400 response for the now since you've not registered using @hameedhub register endpoint

Reply:
Hello @SamuelKoroh, I did all that but I just noticed this morning why it didn't run because I didn't set my `JWTSECRET`. All your test are passing, Good Job. For the swagger documentation, I tested the login endpoint under auth and this what I am getting
<img width="1397" alt="Screen Shot 2019-08-23 at 6 41 06 AM" src="https://user-images.githubusercontent.com/14803275/63569472-6223cb00-c571-11e9-98f8-4a2416aedfa4.png">

Ideally, I should get an error message because I am not a register user, Please fix that.

Reply:
You are using https instead of http protocol